Assets/Scripts/Grid/IGrid.cs

```diff
@@ -32,7 +32,6 @@ public interface IGrid {
         /// TileSize will always be >=1.
         /// </summary>
         uint TileSize { get; }
-        

```

Comment:
Please don't remove this line

Reply:
Actually I see why you did remove it.
I guess it's fine with the comments below.
Let's merge this!